test/controllers/games.js

```diff
@@ -0,0 +1,67 @@
+const chai = require('chai'),
+  dictum = require('dictum.js'),
+  server = require('./../../app'),
+  gameService = require('./../../app/services/games'),
+  sessionManager = require('./../../app/services/sessionManager'),
+  utils = require('./../utils'),
```

Comment:
If you want to avoid all the utils. from below you can do something like this here:
{ successfulLogin, successfulLoginNotAdmin } = require('./../utils')